test/Select.test.tsx

```diff
@@ -49,7 +49,7 @@ describe('Select', () => {
 					render={({ handleSubmit }) => (
 						<form onSubmit={handleSubmit} noValidate data-testid="form">
 							<Select
-								label={label && 'Test'}
+								label={label ? 'Test' : undefined}
```

Comment:
Can you try `!!label && 'Test'`?

Reply:
That would still evaluate to `false` which we don't want

Reply:
Ah... I'm being dumb.